bug-1292234-test-we/background.js

```diff
@@ -0,0 +1,15 @@
+(async function main() {
+
+  browser.storage.local.set({
+    rainbows: "unicorns",
+    catgifs: true,
+  });
+
+  function handleChange(changes, areaName) {
+    if (areaName !== 'local') return;
+    console.log("WebExtensionLog::ExtensionLocalStorageChange", changes);
+  }
+  
+  // Add listeners
```

Comment:
I don't think this comment adds clarity, since it's just describing what is happening on the next line.

If you want to document *why* adding a handler is a good idea here, that would be worth commenting.

Reply:
Good point thanks